extension/ballastextension/README.md

```diff
@@ -1,8 +1,12 @@
+> [!WARNING]
+> The memory ballast extension is deprecated in favor of using the `GOMEMLIMIT` environment variable.
+> This environment variable is available on any Collector built with Go 1.19 or higher. Official binary releases are built with Go 1.19 since v0.61.0. See [issue 8343](https://github.com/open-telemetry/opentelemetry-collector/issues/8343) for the deprecation timeline.
```

Comment:
Should this include a recommendation of what a user should configure this based on their previous setting?